### PR TITLE
updated sqs.md

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/sqs.md [hunk 1]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
-| json "eventName" nodrop
-| where eventSource="sqs.amazonaws.com"
-| parse field=arn ":assumed-role/*" as user
-| if (isEmpty(userName), user, userName) as user
-| count as eventCount by user
-| top 10 user by eventCount
+**Messages Received (Metric based)**:
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
+
+```sql title="Top 10 users (CloudTrail Log based)"
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:72]: Here 'Top 10 users (CloudTrail Log based):' should be present. I don't see it in the md file.

File: docs/integrations/amazon-aws/sqs.md [hunk 2]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
+The Amazon SQS - Threat Intel dashboard provides insights into incoming requests to your AWS SQS services from malicious sources determined via Sumo Logic’s Threat Intel feature. Panels show detailed information on malicious IPs and the malicious confidence of each threat.
 
-**Sent Message Size**. See the average, sum, and maximum of the metric sent message size in the last 24 hours by region on a line chart.
+Use this dashboard to:
+* Monitor details of threat locations and count.
+* Get details of threats by malicious confidence and malicious IPs.
+* Get details of all threats by IPs.
 
-**Lag (Sent - Deleted)**. See the difference between the metrics number of messages sent and number of messages deleted, to get the lag of the messages by region in the last 24 hours on a line chart.
+![Amazon SQS - Threat Intel](https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Amazon-SQS/AmazonSQS-ThreatIntel.png).
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:233]: Remove the dot at the end.

File: docs/integrations/amazon-aws/sqs.md [hunk 3]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
+Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing service that makes it easy to decouple and scale microservices, distributed systems, and serverless applications.
 
-**Number Of Messages Received**. See the sum of the metric number of messages received in the last 24 hours by queue name on a line chart.
+The Sumo Logic App for Amazon SQS provides operational insights into your Amazon SQS utilization. The App’s preconfigured dashboards help you monitor the key metrics, view the SQS events for queue activities, and help you plan the capacity of your SQS service utilization.
 
-**Number Of Messages Sent**. See the sum of the metric number of messages sent in the last 24 hours by queue name on a line chart.
+**Note**: We highly recommend you view these dashboards in the [Explore View](https://help.sumologic.com/Beta/AWS_Observability_Solution/07_View_AWS_Observability_Solution_Dashboards) of the AWS Observability solution.
 
-**Sent Message Size**. See the average, sum, and maximum of the metric sent message size in the last 24 hours by queue name on a line chart.
+###Amazon SQS Overview
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:192]: This heading is not appearing as it should be in md file.

File: docs/integrations/amazon-aws/sqs.md [hunk 4]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
-**Top Users**. See the top 10 users by event count in the last 14 days displayed in a table.
+## Centralized AWS CloudTrail Log Collection
 
-**Events by User**. See the count of SQS events by user in the last 14 days on a bar chart.
+In case you have a centralized collection of CloudTrail logs and are ingesting them from all accounts into a single Sumo Logic CloudTrail log source, create the following **Field Extraction Rule** to map a proper AWS account(s) friendly name/alias. Create it if not already present/update it as required.
 
-**Events Details**. See the details of SQS events including the time, event name, queue name, queue URL, type, user, account ID, AWS region, source IP address, and event count, in the last 14 days, displayed in a table.
+**Rule Name**: AWS Accounts
+**Applied at**: Ingest Time
+**Scope (Specific Data**:_sourceCategory=aws/observability/cloudtrail/logs
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:154]: It should be '**Scope: Specific Data**:_sourceCategory=aws/observability/cloudtrail/logs'

File: docs/integrations/amazon-aws/sqs.md [hunk 5]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
-| json "userIdentity.arn" as arn nodrop
-| json "userName" nodrop
-| json "eventName" nodrop
-| where eventSource="sqs.amazonaws.com"
-| parse field=arn ":assumed-role/*" as user
-| if (isEmpty(userName), user, userName) as user
-| count as eventCount by user
-| top 10 user by eventCount
+**Messages Received (Metric based)**:
+metric=NumberOfMessagesReceived Statistic=Sum account=* region=* namespace=* queuename=* | sum by account, region, namespace, queuename
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:70]: This should be in a new line.

File: docs/integrations/amazon-aws/sqs.md [hunk 6]
```diff
@@ -13,235 +13,221 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
-See the details of SQS metrics including the age, delayed, visible, sent, deleted, messages, and size of the messages.
-
-<img src={useBaseUrl('img/integrations/amazon-aws/SQS-Overview.png')} alt="SQS-Overview" />
+## Field Extraction Rule(s)
 
-**Events Location. **See the count of SQS CloudTrail events by location in the last 14 days on a world map.
+Create a Field Extraction Rule for CloudTrail Logs. Learn how to create a Field Extraction Rule [here](https://help.sumologic.com/Manage/Field-Extractions/Create-a-Field-Extraction-Rule).
 
-**Approximate Age Of Oldest Message (Seconds)**. See the average of the metric approximate age of oldest message in the last 24 hours by region on a line chart.
+**Rule Name**: AwsObservabilitySQSCloudTrailLogsFER
```
Comment (JKashyap96) [inline docs/integrations/amazon-aws/sqs.md:130]: All these should be in new lines.

**Review by JKashyap96 — APPROVED** [review]
LGTM

File: docs/integrations/amazon-aws/sqs.md [hunk 7]
```diff
@@ -13,235 +13,240 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
+**Rule Name**: AWS Accounts
 
-**Events. **See the count and percentage of the different SQS events in the last 14 days on a pie chart.
+**Applied at**: Ingest Time
 
-**Events Trend**. See the trend of different events in the last 14 days on a stacked column chart.
+**Scope (Specific Data)**: _sourceCategory=aws/observability/cloudtrail/logs
 
-**Events Location. **See the count of SQS CloudTrail events by location in the last 14 days on a world map.
+```Parse Expressions
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/sqs.md:167]: same comment as https://github.com/SumoLogic/sumologic-documentation/pull/814/files#r1009679725

**Review by kimsauce — CHANGES_REQUESTED** [review]
two changes

File: docs/integrations/amazon-aws/sqs.md [hunk 8]
```diff
@@ -13,235 +13,240 @@ Amazon Simple Queue Service (Amazon SQS) is a fully managed message queuing serv
...
-**Lag (Sent - Deleted).** See the difference between the metrics number of messages sent and number of messages deleted, to get the lag of the messages by region in the last 24 hours on a line chart.
-
-**Number Of Messages Received**. See the sum of the metrics number of messages received by region in the last 24 hours on a line chart.
-
-**Sent Message Size**. See the average, sum, and maximum of the metric sent message size in the last 24 hours by region on a line chart.
-
-**Number Of Messages Sent and Deleted**. See the sum of the metric number of messages sent, and the sum of the metric number of messages deleted, in the last 24 hours by region on a line chart.
+**Scope (Specific Data)**: account=* eventname eventsource "sqs.amazonaws.com"
 
+```Parse Expression
```
[the remark this answers is not in the document]
Comment (JKashyap96) [reply docs/integrations/amazon-aws/sqs.md]: Please check now.

File: docs/integrations/amazon-aws/sqs.md (hunk 7 above)
Comment (JKashyap96) [reply docs/integrations/amazon-aws/sqs.md]: Please check now.

**Review by kimsauce — APPROVED** [review]

**Comment by JKashyap96** [comment]
@kimsauce Will you be merging this?

**Comment by JainM6** [comment]
@Jayanta Kashyap ***@***.***> : I have added changes to the
file as per the review suggestions, so this can't be merged until I commit
the new changes.

**Comment by kimsauce** [comment]
> @kimsauce Will you be merging this?

@JainM6 will be

**Pull request merged**